src/web/components/ParticipantManagement/UpdateParticipantDialog.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react';
+import { SubmitHandler } from 'react-hook-form';
+
+import { ApiRoleDTO } from '../../../api/entities/ApiRole';
+import { ParticipantDTO } from '../../../api/entities/Participant';
+import { UpdateParticipantForm } from '../../services/participant';
+import { Dialog } from '../Core/Dialog';
+import { Form } from '../Core/Form';
+import { CheckboxInput } from '../Input/CheckboxInput';
+
+type UpdateParticipantDialogProps = {
+  triggerButton: JSX.Element;
+  participant: ParticipantDTO;
+  onUpdateParticipant: (form: UpdateParticipantForm, participant: ParticipantDTO) => Promise<void>;
+  apiRoles: ApiRoleDTO[];
+};
+
+function UpdateParticipantDialog({
+  apiRoles,
+  onUpdateParticipant,
+  participant,
+  triggerButton,
+}: UpdateParticipantDialogProps) {
+  const [open, setOpen] = useState(false);
+
+  const onSubmit: SubmitHandler<UpdateParticipantForm> = async (formData) => {
+    await onUpdateParticipant(formData, participant);
+    setOpen(false);
+  };
+
+  const originalFormValues: UpdateParticipantForm = {
+    apiRoles: participant.apiRoles ? participant.apiRoles.map((apiRole) => apiRole.id) : [],
+  };
+
+  return (
+    <Dialog
+      triggerButton={triggerButton}
+      title={`Edit ${participant.name}`}
+      closeButton='Cancel'
+      open={open}
+      onOpenChange={setOpen}
+    >
+      <Form<UpdateParticipantForm>
+        onSubmit={onSubmit}
+        submitButtonText='Update Participant'
```

Comment:
I think this should probably be 'Save Participant' for consistency with other edit dialogs e.g. TeamMember and BusinessContact